src/cmd_line/commands/file.ts

```diff
@@ -85,6 +86,19 @@ export class FileCommand extends node.CommandBase {
       let folder = vscode.Uri.file(newFilePath);
       await vscode.commands.executeCommand("vscode.open", folder,
         this._arguments.position === FilePosition.NewWindow ? this.getViewColumnToRight() : this.getActiveViewColumn());
+
+      await vscode.commands.executeCommand("cursorMove", {
```

Comment:
How do we handle the :50 command? This should be done the same way, this hack should not be necessary for sure... There should also be a vscode command to go to line